cpp/src/parquet/metadata.cc

```diff
@@ -449,6 +453,17 @@ std::unique_ptr<ColumnChunkMetaData> RowGroupMetaData::ColumnChunk(int i) const
   return impl_->ColumnChunk(i);
 }
 
+bool RowGroupMetaData::can_decompress() const {
+  int n_columns = num_columns();
+  for (int i = 0; i < n_columns; i++) {
+    auto column = ColumnChunk(i);
+    if (column == nullptr || !column->can_decompress()) {
```

Comment:
What would `column == nullptr` mean here?

Reply:
I put it there defensively agaisn't bad inputs, but looking at the implementation it'll raise an exception/segfault even before we get this case. Want me to remove them?

Reply:
Well, if a NULL column doesn't have a well-defined meaning, it should certainly error out before, so it's confusing to see this here.

Reply:
So, yes, I'd remove have them removed.